magento/pwa-studio
# Fix/update cms component with meta data

## Description

This PR updates the cms.js component to make use of `data.cmsPage.title` and `data.cmsPage.content_heading`, that otherwise seems to be ignored.

I have placed the `<Title>` and `<h1>` in this location so that it doesn't affect the home page. I am by no means saying this is the perfect place to put it, I'm very open to suggestions of where to move these elements, but I think they should be included _somewhere_.
I have attached a couple of screenshots, showing an "about-us" page with and without this update. In the "before" screenshot, we can see how hard it is to determine which page we're on, especially if we have multiple tabs open on the storefront.

The classname given to the `<h1>` is also a bit of a guess, so I'm open to updating this to better suit.

## Related Issue
Closes #2158.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps
1. Create a new page
2. Give the new page a content heading, and some content
3. Navigate to the URL slug of the new page using a PWA instance
4. Verify that the page title updates to reflect the name of the page that you're on, and that the content heading that was entered into admin is also displaying

## Screenshots / Screen Captures (if appropriate)
**Before Update**
![image](https://user-images.githubusercontent.com/18253422/74210784-127eba80-4cdd-11ea-8cda-345789db5181.png)

**After Update**
![image](https://user-images.githubusercontent.com/18253422/74210655-781e7700-4cdc-11ea-9ef8-970e6d8d940b.png)
